pages/api/admin/reset.js

```diff
@@ -65,23 +65,34 @@ async function deleteVisitorIdData(visitorData) {
     where: { visitorId: visitorData.visitorId },
   };
 
-  const loginAttemptsRowsRemoved = await LoginAttempt.destroy(options);
+  const loginAttemptsRowsRemoved = await tryToDestroy(() => LoginAttempt.destroy(options));
 
-  const paymentAttemptsRowsRemoved = await PaymentAttempt.destroy(options);
+  const paymentAttemptsRowsRemoved = await tryToDestroy(() => PaymentAttempt.destroy(options));
 
-  const couponsRemoved = await CouponClaim.destroy(options);
+  const couponsRemoved = await tryToDestroy(() => CouponClaim.destroy(options));
 
   const deletedPersonalizationResult = await Promise.all(
-    [UserCartItem, UserPreferences, UserCartItem, UserSearchHistory].map((model) => model.destroy(options))
+    [UserCartItem, UserPreferences, UserCartItem, UserSearchHistory].map((model) =>
+      tryToDestroy(() => model.destroy(options))
+    )
   );
+
   const deletedPersonalizationCount = deletedPersonalizationResult.reduce((acc, cur) => acc + cur, 0);
-  const deletedLoanRequests = await LoanRequest.destroy(options);
 
-  const deletedPaywallData = await ArticleView.destroy(options);
+  const deletedLoanRequests = await tryToDestroy(() => LoanRequest.destroy(options));
+  const deletedPaywallData = await tryToDestroy(() => ArticleView.destroy(options));
 
   return new CheckResult(
     `Deleted ${loginAttemptsRowsRemoved} rows for Credential Stuffing problem. Deleted ${paymentAttemptsRowsRemoved} rows for Payment Fraud problem. Deleted ${deletedPersonalizationCount} entries related to personalization.  Deleted ${deletedLoanRequests} loan request entries. Deleted ${deletedPaywallData} rows for the Paywall problem. Deleted ${couponsRemoved} rows for the Coupon fraud problem.`,
     messageSeverity.Success,
     checkResultType.Passed
   );
 }
+
+const tryToDestroy = async (callback) => {
+  try {
+    return await callback();
+  } catch (err) {
+    return 0;
```

Comment:
What are the possible reasons for which errors will be thrown? Maybe we could at least log it into the console?

Reply:
Reason is mostly the table has not created yet, just pushed a new commit for logging error